BUILD.md

```diff
@@ -59,11 +59,11 @@ list some basic and more used commands as example.
     ```sh
     ./waf configure --board bebop --static # Bebop or Bebop2
     ./waf configure --board edge           # emlid edge
-    ./waf configure --board fmuv3          # Pixhawk2/Cube using ChibiOS
+    ./waf configure --board fmuv3          # Pixhawk 1 using ChiBiOS
     ./waf configure --board fmuv4          # Pixracer using ChibiOS
     ./waf configure --board navio2         # emlid navio2
     ./waf configure --board Pixhawk1       # Pixhawk1
-    ./waf configure --board CubeBlack      # Pixhawk2
+    ./waf configure --board CubeBlack      # Pixhawk2/CubeBlack using ChiBiOS
```

Comment:
This should be `# Hex/ProfiCNC Cube Black  Only`

Reply:
I did raise this question on the [forum](https://discuss.cubepilot.org/t/fmuv3-vs-cubeblack-firmware/876/6), asking about whether the new Cube Blacks and the older Pixhawk2/Cube should be treated differently. From what I infer from Rowse's reply, it seems that all Cubes should use the Cube Black firmware. 

@proficnc appreciate your help to clear the confusion about this (:

Reply:
The "pixhawk 2.1 cube" that was made for and used in the 3DR Solo is a _cube shaped Pixhawk 2_ manufactured by 3DR.  **It is NOT a Hex manufactured Cube Black**.  The hardware is different, and it cannot use the Cube Black build target.  It uses FMUv3 only. The only products that can use the Cube Black build target are actual Hex manufactured products called Cube Black.  This is per Phillip.

FYI, if you put a Cube Black firmware build onto anything but a real Cube Black, it will fail on boot with sensor errors.

Reply:
I think I understand now. So to paraphrase what has been said:

- If it is a Hex/Proficnc manufactured Cube Black/Pixhawk 2, then it requires the Cube Black firmware (regardless of which batch/year the Cube came from)

- If it is the old Pixhawk 2.1 manufactured by 3DR, then it can only run on fmuv3

Please correct me if I am still not getting it right.

Reply:
No. The Pixhawk 2.1 is not a Hex product. Please see above where I wrote exactly what it needs to be.  You can just copy and paste what I suggested the comments say instead and that will be accurate.

Reply:
Matt all black cubes was 2.1 3dr was 2